src/compute-model-statistics/src/test/scala/VerifyComputeModelStatistics.scala

```diff
@@ -240,4 +241,32 @@ class VerifyComputeModelStatistics extends TestBase {
     assert(auc === cmsAUC)
   }
 
+  override def testObjects(): Seq[TestObject[ComputeModelStatistics]] = Seq(new TestObject(
+    new ComputeModelStatistics(),
+    {
+      val labelColumn = "label"
+      val predictionColumn = SchemaConstants.SparkPredictionColumn
+      val labelsAndPrediction = session.createDataFrame(
```

Comment:
maybe move the multiclass metrics dataset out and reuse it here and in the multiclass metrics test (this dataframe appears twice in the test file it looks like)

Reply:
Done

Reply:
it looks like it is still not reused (?)

Reply:
ah, you haven't pushed a new commit yet, will wait for it...